src/main/scala/org/broadinstitute/dsde/firecloud/service/EntityService.scala

```diff
@@ -0,0 +1,78 @@
+package org.broadinstitute.dsde.firecloud.service
+
+/**
+ * Created by mbemis on 7/10/15.
+ */
+
+import javax.ws.rs.Path
+
+import akka.actor.{Actor, Props}
+import com.wordnik.swagger.annotations._
+import org.broadinstitute.dsde.firecloud.{EntityClient}
+import org.broadinstitute.dsde.firecloud.model.{WorkspaceEntity, WorkspaceIngest}
+import org.broadinstitute.dsde.firecloud.model.ModelJsonProtocol._
+import org.slf4j.LoggerFactory
+import spray.http.StatusCodes._
+import spray.httpx.SprayJsonSupport._
+import spray.routing._
+import org.broadinstitute.dsde.vault.common.directives.OpenAMDirectives._
+
+class EntityServiceActor extends Actor with EntityService {
+  def actorRefFactory = context
+  def receive = runRoute(routes)
+}
+
+@Api(value = "/workspaces/{workspaceNamespace}/{workspaceName}/entities", description = "Entity Service", position = 2, produces = "application/json")
+trait EntityService extends HttpService with FireCloudDirectives {
+
+  private final val ApiPrefix = "entities"
+  private implicit val executionContext = actorRefFactory.dispatcher
+
+  val routes =  optionsRoute ~
+                listEntitiesPerTypeRoute
+
+  lazy val log = LoggerFactory.getLogger(getClass)
+
+  @ApiOperation(
+    value = "entities options",
+    nickname = "entitiesOptions",
+    httpMethod = "OPTIONS",
+    notes = "response is an OPTIONS response")
+  @ApiResponses(Array(
+    new ApiResponse(code = 200, message = "OK")))
+  def optionsRoute: Route =
+    path(ApiPrefix) {
```

Comment:
I'm not following this. How is the application handling "/entities" but the swagger @Api is using "/workspaces/{workspaceNamespace}/{workspaceName}/entities"? Doesn't this route need to handle "/entities" as a path suffix of some sort, along with a path prefix of "/workspace"?